ouroboros-consensus/src-win32/Ouroboros/Storage/IO.hs

```diff
@@ -80,3 +83,17 @@ close fh = closeHandleOS fh closeHandle
 getSize :: FHandle -> IO Word64
 getSize fh = withOpenHandle "getSize" fh $ \h -> do
   fromIntegral . bhfiSize <$>  getFileInformationByHandle h
+
```

Comment:
Let's leave a comment here. Basically, what this means is that for `FsReachedEOF`, `FsDeviceFull` and `FsResourceAlreadyInUse`, our mock implementation (and indeed Posix) both throw the same error as Windows does, but for the rest Windows throws different errors (currently more details unknown).